fastscanner/services/indicators/lib/candle.py

```diff
@@ -255,6 +255,69 @@ def column_name(self):
     def lookback_days(self) -> int:
         return 0
 
+    async def extend(self, symbol: str, df: pd.DataFrame) -> pd.DataFrame:
+        df = await self._high_n_days.extend(symbol, df)
+        df = await self._low_n_days.extend(symbol, df)
+
+        high_col = self._high_n_days.column_name()
+        low_col = self._low_n_days.column_name()
+
+        df[self.column_name()] = (df[CandleCol.CLOSE] - df[low_col]) / (
+            df[high_col] - df[low_col]
+        )
+
+        return df.drop(columns=[high_col, low_col])
```

Comment:
This creates a bug because if the column high_col and low_col already existed in the df, it will drop them anyway.
You need to check if the original df had the columns and only drop them if they were not there already.